ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/jline/JlineTerminalAdapter.java

```diff
@@ -56,10 +56,12 @@ public String readLine(String prefix, String postfix) throws ShellExitException
         }
     }
 
+    @SuppressWarnings("resource")
     @Override
     public void println(String text) {
-        lineReader.getTerminal().writer().println(text);
-        lineReader.getTerminal().writer().flush();
+        var writer = lineReader.getTerminal().writer();
```

Comment:
As a practice, we are not using `var` keyword. Use the Class name here.

Reply:
@hasithaa 
Does that also apply to cases where the variable is declared inside a try with resource?
Example:
```java
try (var paths = Files.walk(src)) {
    paths.forEach(source -> copy(source, dest.resolve(src.relativize(source))));
}
```